lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -289,7 +297,19 @@ def self.available_options
                                   env_name: "APPCENTER_DISTRIBUTE_RELEASE_NOTES_LINK",
                                description: "Additional release notes link",
                                   optional: true,
-                                      type: String)
+                                      type: String),
+
+          FastlaneCore::ConfigItem.new(key: :build_number,
+                                       env_name: "APPCENTER_DISTRIBUTE_BUILD_NUMBER",
+                                       description: "Android apk build number",
+                                       optional: true,
+                                       type: String),
+
+          FastlaneCore::ConfigItem.new(key: :version,
+                                       env_name: "APPCENTER_DISTRIBUTE_VERSION",
+                                       description: "Android apk version name",
```

Comment:
Same here. Would suggest change this to `The version number. Used (and required) for uploading Android ProGuard mapping file.`